### PR TITLE
63483: Document new S3 Credential checkboxes (master)

### PR DESCRIPTION
**Pull request by Mrt134**
- Add entries to Cloud Credential Options table in system.rst
- Add entry to intro.rst U2 section.
- HTML build test: no issues.

### CODE REVIEW

File: userguide/intro.rst [hunk 1]
```diff
@@ -357,7 +357,7 @@ U2
 * The :guilabel:`Endpoint does not support regions` and
```
Comment (wblock) [inline userguide/intro.rst]: I thought we changed that, please verify field label is correct here.

File: userguide/intro.rst [hunk 2]
```diff
@@ -357,7 +357,7 @@ U2
...
   :guilabel:`Use v2 signatures` fields have been added to
```
Comment (wblock) [inline userguide/intro.rst]: Please verify field label text and capitalization.

File: userguide/intro.rst (hunk 1 above)
Comment (Mrt134) [reply userguide/intro.rst]: Rechecked: it is correct for the legacy UI

File: userguide/intro.rst (hunk 2 above)
Comment (Mrt134) [reply userguide/intro.rst]: Rechecked: it is correct for the legacy UI

**Pull request merged**